NuGet.Config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Our testing infrastructure depends on a known location for:
+     - NUnit.Console
+     - csc.exe
+-->
+<configuration>  
+  <config>
+    <add key="globalPackagesFolder" value="$\..\packages" />
```

Comment:
What *is* `$` in `$\..\packages`?  I can't easily find any mention of this behavior.  ([Environment variables are supported](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#using-environment-variables).)

Reply:
Apparently it is "local to *this* NuGet.Config", since you could have several configs getting inherited.

```
For the repositoryPath setting, you can specify an absolute path or relative path (recommended) using the $ token. The $ token is based on where the NuGet.Config is located (The $ token is actually relative to one level below the location of the NuGet.Config). So, if I have \Solutions\NuGet.Config and I want \Solutions\Packages I would need to specify $\..\Packages as the value.
```

https://stackoverflow.com/questions/18376313/setting-up-a-common-nuget-packages-folder-for-all-solutions-when-some-projects-a